deps/juggler-v3/test.js

```diff
@@ -0,0 +1,30 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: loopback-connector-db2
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const should = require('should');
+const juggler = require('loopback-datasource-juggler');
+
+require('../../test/init');
+
+describe('db2 imported feature v3', function() {
+  before(function() {
+    return global.resetDataSourceClass(juggler.DataSource);
+  });
+
+  after(function() {
+    return global.resetDataSourceClass();
+  });
+
+  require('loopback-datasource-juggler/test/common.batch.js');
+  require('loopback-datasource-juggler/test/include.test.js');
+
+  // const suite = require("loopback-datasource-juggler/test/persistence-hooks.suite.js");
+  // const customConfig = Object.assign({}, global.config);
+  // suite(global.getDataSource(customConfig, juggler.DataSource), should, {
+  //   replaceOrCreateReportsNewInstance: false
+  // });
```

Comment:
Is this code needed since it's commented out (same for v4)?

Reply:
I think it might have the same issue as Postgres: https://github.com/strongloop/loopback-connector-postgresql/pull/376/files#diff-462e77db1e3bbf1f78504e62025b2ffbR24. We will get this error:
```
09:34:25   1) db2 imported feature v3
09:34:25        Persistence hooks
09:34:25          "before each" hook: setupDatabase for "triggers hooks in the correct order":
09:34:25      Error: [IBM][CLI Driver][DB2/LINUXX8664] SQL0204N  "POINT" is an undefined name.  SQLSTATE=42704
09:34:25 
09:34:25   
09:34:25 
09:34:25   2) db2 imported feature v4
09:34:25        Persistence hooks
09:34:25          "before each" hook: setupDatabase for "triggers hooks in the correct order":
09:34:25      Error: [IBM][CLI Driver][DB2/LINUXX8664] SQL0204N  "POINT" is an undefined name.  SQLSTATE=42704
```